util/hello/hello.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
+
+from termcolor import colored
+
```

Comment:
Would be interesting to see if you can use `opentitanlib.*` modules in here?

Reply:
Yes it works no problem. I have added an example in the last update